core/src/main/resources/lib/form/apply_pl.properties

```diff
@@ -1,3 +0,0 @@
-# This file is under the MIT License by authors
```

Comment:
Are all the `apply*.properties` files obsolete? If so, and in case you need to update this PR anyway, could you delete the others?